examples/SetBucketNotification.java

```diff
@@ -48,7 +48,7 @@ public static void main(String[] args)
       NotificationConfiguration config = new NotificationConfiguration();
 
       // Add a new SQS configuration.
-      List<QueueConfiguration> queueConfigurationList = config.queueConfigurationList();
+      List<QueueConfiguration> queueConfigurationList = new LinkedList<>(config.queueConfigurationList());
```

Comment:
just do `List<QueueConfiguration> queueConfigurationList = new LinkedList<>();`

Reply:
copy that